README.md

```diff
@@ -163,6 +164,10 @@ Pull requests are welcome 👍. For major changes, please open an issue first to
 ------------------------------------
 ## Contributors ✨
 
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification for recognizing project contributors. 
+
+Contributions of any kind welcome!
```

Comment:
reviewer comment: I think this is better located on top of the section. Let me know if I need to revert this.